README.md

```diff
@@ -15,7 +15,8 @@ ssh leonardo
 git clone XXX
 VENV_DIR=~/llmeval/venv/  # where you want to setup a virtual env
 HF_HOME=~/llmeval/hf/  # where you want to download HF datasets
-bash llmeval/setup_node.sh $VENV_DIR $HF_HOME
+HF_TOKEN=<your huggingface API token>
```

Comment:
We should not ask user to put token in code, they are bound to appear in PR, branch etc.
A better alternative would be to use `source` instead of `bash` which would run everything without spawning a subprocess, we can then use the environment variable defined when calling setup_node.sh.